docs/content/concepts/sui-move-concepts/conventions.mdx

```diff
@@ -0,0 +1,493 @@
+---
+title: Conventions
+description: Recommended Move 2024 best practices for the Sui blockchain.
+---
+
+The following recommendations are based on 2024 Move.
+
+## Add section titles
+
+Use titles in code comments to create sections for your Move code files. Structure your titles using `===` on either side of the title.
+
+```move
+module conventions::comments {
+  // === Imports ===
+
+  // === Friends ===
+
+  // === Errors ===
+
+  // === Constants ===
+
+  // === Structs ===
+
+  // === Public-Mutative Functions ===
+
+  // === Public-View Functions ===
+
+  // === Admin Functions ===
+
+  // === Public-Friend Functions ===
+
+  // === Private Functions ===
+
+  // === Test Functions ===
+}
+```
+
+## CRUD functions names
+
+These are the available CRUD functions:
+
+- `add`: Adds a value.
+- `new`: Creates an object.
+- `drop`: Drops a struct.
+- `empty`: Creates a struct.
+- `remove`: Removes a value.
+- `exists_`: Checks if a key exists.
+- `contains`: Checks if a collection contains a value.
+- `destroy_empty`: Destroys an object or data structure that has values with the **drop** ability.
+- `to_object_name`: Transforms an Object X to Object Y.
+- `from_object_name`: Transforms an Object Y to Object X.
+- `property_name`: Returns an immutable reference or a copy.
+- `property_name_mut`: Returns a mutable reference.
+
+## Potato structs
+
+Do use 'potato' in the name of structs. The lack of abilities define it as a potato pattern.
```

Comment:
I believe this was a typo in @ronny-mysten's review. The text should still say `don't`.

Reply:
ooooops